cypress/support/step-definitions/select-steps.js

```diff
@@ -225,6 +226,12 @@ Then("Select input has no value", () => {
   getDataElementByValue("input").should("have.attr", "value", "");
 });
 
+Then("{string} Select list is visible at the {word}", (name, position) => {
+  selectListPosition(name)
+    .should("have.attr", "data-popper-placement", position)
```

Comment:
and the same is here:
```suggestion
    .should("have.attr", "data-popper-placement", `${position}-end`)
```